shishito/runtime/platform/shishito_control_test.py

```diff
@@ -80,3 +80,6 @@ def test_init(self, url):
 
         :param str url:
         """
+        self.driver.get(url)
+        self.driver.implicitly_wait(int(self.shishito_support.get_opt('default_implicit_wait')))
+
```

Comment:
Maybe this should go to the more specific "mobile" version of **control_test** (_\shishito\runtime\platform\mobile\control_test.py_), not directly to the parent class (might have some effects on the execution on other platforms. But I am not 100% sure just by looking into the code.)